content/engineering/bruteforce-lock-and-unlock/index.md

```diff
@@ -0,0 +1,174 @@
+    - [Testing Brute force lockout](#testing-brute-force-lockout)
+    - [Lockout types supported by LoginRadius](#lockout-types-supported-by-loginradius)
+    - [Unlocking an account locked through brute force lockout](#unlocking-an-account-locked-through-brute-force-lockout)
+        - [Account Update API from the LoginRadius Account API collections](#account-update-api-from-the-loginradius-account-api-collections)
+        - [Auth Unlock Account by Access Token from the LoginRadius Authentication API collections](#auth-unlock-account-by-access-token-from-the-loginradius-authentication-api-collections)
+- [Conclusion](#conclusion)
+- [Miscellaneous](#miscellaneous)
+    - [CAPTCHA](#captcha)
+    - [MFA](#mfa)
+
+---
+
+## Basic Terminologies
+1. **Brute force attack:**
+- A method where every possible combination of characters or values is systematically tried to gain unauthorized access to a system, application, or data.
+
+2. **Brute force lock:**
+- Brute force lock is a type of account lock made to prevent a bruteforce attack.
+
+3. **Brute force lockout:**
+- Brute force lockout is a security mechanism that blocks access after a certain number of failed authentication attempts to prevent unauthorized access through repeated trial and error.
+
+4. **CAPTCHA (Completely Automated Public Turing test to tell Computers and Humans Apart):**
+- CAPTCHA is a method used to determine whether a user is human or not by presenting a challenge that is easy for humans to solve but difficult for bots.
+
+5. **Multi-Factor Authentication (MFA):**
+-  MFA is a security method that requires multiple forms of identification to grant access, typically combining something the user knows, has, and is.
+
+In _LoginRadius_, the brute force lockout can be implemented in two ways, using:
+1. APIs
+2. Workflows
+
+> To implement brute force lockout in either of the above-mentioned ways, the user should own an admin account in the [_LoginRadius Admin Console_](https://admin-console.loginradius.com/dashboard).
+
+In this blog, we'll go through the API implementation of brute force lockout and user unlock
+
+---
+
+## API implementation
+### Creating a Basic Application
+1. In order to achieve brute force lockout using API, we need create a simple app with the login and registration features.
+2. This can be done by using the _Admin Console_.
+3. Navigate through _[Deployment>Identity Experience Framework](https://devadmin-console.lrinternal.com/deployment/idx)_.
+4. Using theme, customization, preview, implement options, we can design the required application.
+5. We can customize the predefined templates if needed.
+
+> The created app can be accessed using the link _[https://`<app-name>`.hub.loginradius.com/auth.aspx](https://`<app-name>`.hub.loginradius.com/auth.aspx)_ in the _implement_ section of the _Identity Experience Framework_ or can be accessed from the preview section.
+
+### Testing Brute force lockout
```

Comment:
@gayathrisuresh150501 We can mention the steps from where we can enable the brute force lockout in our admin console.